xrdp/xrdp.ini

```diff
@@ -1,43 +1,57 @@
-[globals]
-# xrdp.ini file version number
+[Globals]
+; xrdp.ini file version number
 ini_version=1
 
-bitmap_cache=yes
-bitmap_compression=yes
+; fork a new process for each incoming connection
+fork=true
+; tcp port to listen
 port=3389
-allow_channels=true
-max_bpp=32
-fork=yes
-# minimum security level allowed for client
-# can be 'none', 'low', 'medium', 'high', 'fips'
-crypt_level=high
-# security layer can be 'tls', 'rdp' or 'negotiate'
-# for client compatible layer
+; regulate if the listening socket use socket option tcp_nodelay
+; no buffering will be performed in the TCP stack
+tcp_nodelay=true
+; regulate if the listening socket use socket option keepalive
+; if the network connection disappear without close messages the connection will be closed
+tcp_keepalive=true
+#tcp_send_buffer_bytes=32768
+#tcp_recv_buffer_bytes=32768
+
+; security layer can be 'tls', 'rdp' or 'negotiate'
+; for client compatible layer
 security_layer=negotiate
-# X.509 certificate and private key
-# openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem -days 365
+; minimum security level allowed for client
+; can be 'none', 'low', 'medium', 'high', 'fips'
+crypt_level=high
+; X.509 certificate and private key
+; openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem -days 365
 certificate=
 key_file=
-# disable SSlv3
-#disableSSLv3=yes
-# set TLS cipher suites
+; regulate xrdp if to accept SSLv3 connections
```

Comment:
"specify whether SSLv3 should be disabled"

Aside, we should avoid all "negative" options, they require a mental negation. "enableSSLv3" would have been better.